tasks/publish-to-mrrc/publish-to-mrrc.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
```

Comment:
Can you add the labels and annotations like https://github.com/konflux-ci/release-service-catalog/blob/development/tasks/add-fbc-contribution/add-fbc-contribution.yaml#L6?

Reply:
Sure, will fix.

Reply:
Fixed.